docs/install_guide.rst

```diff
@@ -105,21 +119,7 @@ present at the root of the repository. ::
 
   ./kong.config.sh
 
-User creation
-^^^^^^^^^^^^^
-
-The `admin` user is automatically created and have complete access privilege.
-Admin default password is 'admin'. Change this password ASAP.
-One can easily change password and create new users using the web interface.
-Login as admin and navigate to the 'Auth' menu.
-
-
-Access policies
-^^^^^^^^^^^^^
-
-Dojot use a RBAC system to manage authorization.
-Two groups are created automatically:
-- Admin: full privilege group
-- User: Can access everything, except manage user and permissions.
+.. note::
 
-Soon, a interface to manage groups and permissions will be available.
+  On previous iterations of dojot, two extra installation steps were required: creating an 'admin'
+  user and configuring default access policies. Those are no longer necessary.
```

Comment:
On this last piece of text, the scripts to create users and access policies is removed, won't it lead to problems if the user select a release, like he was told to do in previous parts of the text, and that specific version requires this scripts?

Reply:
Wouldn't it be better if we remove this last paragraph?